airbyte-integrations/connectors/destination-snowflake-cortex/destination_snowflake_cortex/config.py

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+#
+
+
+from airbyte_cdk.destinations.vector_db_based.config import VectorDBConfigModel
+from pydantic import BaseModel, Field
+
+
+# to-do - override the embedding model to use Snowflake supported model
+class SnowflakeCortexIndexingModel(BaseModel):
+    account: str = Field(
+        ...,
+        title="Account",
+        airbyte_secret=True,
+        description="Enter the account name you want to use to access the database.",
+        examples=["xxx.us-east-2.aws"],
+    )
```

Comment:
I compared these with the Java-based snowflake destination to see if these parameters match the similar destination. If we can make the auth/config flow consistent between the two, this could help end users who might end up using both for different use cases.

It looks like all of our specified inputs match _except_ account, which the Java Snowflake destination calls `host` ("Host").

Also, it looks like we are not accepting `schema` ("Default Schema") and we should probably add that so users can control which schema is written to.

The last difference I noted was that the way we are configuring password is _slightly_ different, just because the Java destination supports more auth options (OAuth and Key Pair private key). Even if we just have a single auth option, it probably makes sense to try to mirror the same structure, which is to place "password" under a "credentials" object. The challenge here is that we can't necessarily copy code from the Snowflake source - since it is in Java, but maybe another Python connector that supports multiple auth flows could be a model here.

Here is the `spec.json` for `destination-snowflake`: https://github.com/airbytehq/airbyte/blob/72b083cbd259b3d922e522bee70e81d60f88f481/airbyte-integrations/connectors/destination-snowflake/src/main/resources/spec.json

Reply:
I will take a look at this tomm.

Reply:
Agreed! The parameters should ideally match the snowflake destination. Made the following changes: 
* replaced host with account. 
* reordered params to match the order in snowflake destination
* added default_schema field
* added a new credentials section with password field.
Implementing the Oauth option seems to much at the moment, but above changes should help us do so in future.